docs/sources/dashboards/variables/add-template-variables/index.md

```diff
@@ -326,27 +336,27 @@ Now you can [filter data on the dashboard](ref:filter-dashboard).
 
 Interpolating a variable with multiple values selected is tricky as it is not straight forward how to format the multiple values into a string that is valid in the given context where the variable is used. Grafana tries to solve this by allowing each data source plugin to inform the templating interpolation engine what format to use for multiple values.
 
-{{% admonition type="note" %}}
+{{< admonition type="note" >}}
 The **Custom all value** option on the variable must be blank for Grafana to format all values into a single string. If it is left blank, then Grafana concatenates (adds together) all the values in the query. Something like `value1,value2,value3`. If a custom `all` value is used, then instead the value is something like `*` or `all`.
-{{% /admonition %}}
+{{< /admonition >}}
 
 #### Multi-value variables with a Graphite data source
 
 Graphite uses glob expressions. A variable with multiple values would, in this case, be interpolated as `{host1,host2,host3}` if the current variable value was _host1_, _host2_, and _host3_.
 
 #### Multi-value variables with a Prometheus or InfluxDB data source
 
-InfluxDB and Prometheus use regex expressions, so the same variable would be interpolated as `(host1|host2|host3)`. Every value would also be regex escaped. If not, a value with a regex control character would break the regex expression.
+InfluxDB and Prometheus use regular expressions, so the same variable would be interpolated as `(host1|host2|host3)`. Every value would also be regular expression escaped. If not, a value with a regular expression control character would break the regular expression.
```

Comment:
Not for this PR but the following sentences are kinda odd. They don't really use the language I'd expect to see when talking about escaping regular expression characters which typically talks of [metacharacters](https://en.wikipedia.org/wiki/Metacharacter) rather than "control characters". I've also never heard the term "regex escaped" ("regular expression escaped" post-Vale).

Reply:
I'll keep that in mind for when I do a revamp of these docs. Probably next quarter.